src/data/faq.json

```diff
@@ -694,7 +694,7 @@
                         "anchor": "mobile_data_costs",
                         "active": true,
                         "textblock": [
-                            "The German network operators have committed themselves to a so-called zero rating. This means that the mobile data volume for the use of the Corona-Warn-App is provided free of charge both in Germany and within the EU. Mobile data that are transmitted via Corona-Warn-App when diagnosis keys are uploaded or when the server notifies you in case of exposure is free of charge and will not be deducted from your data plan.",
+                            "The German network operators have committed themselves to a so-called zero rating. This means that the mobile data volume for the use of the Corona-Warn-App is provided free of charge both in Germany and within the EU. Mobile data that is transmitted via Corona-Warn-App when diagnosis keys are uploaded or when the server notifies you in case of exposure is free of charge and will not be deducted from your data plan.",

```

Comment:
"via Corona-Warn-App" =>
"via the Corona-Warn-App"

Reply:
fixed :)